### PR TITLE
Allow a chained source to continue processing it's authentication source

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Allow a chained source to continue processing it's authentication source
## Impacts

The portal workflow for a chained source.
## NEWS file entries

New Features
++++++++++++
- Chained sources can now continue processing it's authentication source in addition to it's chained source
## Delete branch after merge

NO

### CODE REVIEW

File: html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Chained.pm [hunk 1]
```diff
@@ -33,6 +33,11 @@ has_field 'authentication_source' =>
    options_method => \&options_authentication_source,
   );
 
+has_field authentication_source_can_continue => (
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Chained.pm]: I'm really not sure what this does...
A help message would be really useful here

Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Chained.pm]: If it is enabled then the source defined by the authentication_source attribute can be used in addition to the registered user.

File: html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm [hunk 2]
```diff
@@ -259,6 +260,32 @@ sub checkIfChainedAuth : Private {
     }
 }
 
+=head2 continue_chained_auth
```
Comment (julsemaan) [inline html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm]: skipChainedSource would be a better name I think

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
I'm not sure about what this does and if it's really useful for a use case.

Is this used somewhere or requested by a user ?

**Pull request closed without merging**